rbc_base/MotorSetpoint[3] setpoints